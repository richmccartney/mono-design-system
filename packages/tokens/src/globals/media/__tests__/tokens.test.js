const mediaTokens = require('../tokens.json');

describe('media tokens', () => {
  test(': matches snapshot', () => {
    expect(mediaTokens).toMatchSnapshot();
  });
});
