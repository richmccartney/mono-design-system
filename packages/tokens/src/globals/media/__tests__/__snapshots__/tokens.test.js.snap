// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`media tokens : matches snapshot 1`] = `
Object {
  "media": Object {
    "max-width": Object {
      "lg": Object {
        "value": "1280px",
      },
      "md": Object {
        "value": "1024px",
      },
      "sm": Object {
        "value": "768px",
      },
      "xs": Object {
        "value": "480px",
      },
    },
    "min-width": Object {
      "lg": Object {
        "value": "1024px",
      },
      "md": Object {
        "value": "768px",
      },
      "sm": Object {
        "value": "480px",
      },
      "xs": Object {
        "value": "0",
      },
    },
    "prefers-color-scheme": Object {
      "dark": Object {
        "value": "dark",
      },
      "light": Object {
        "value": "light",
      },
    },
    "prefers-reduced-motion": Object {
      "value": "no-preference",
    },
  },
}
`;
